composer.json

```diff
@@ -20,7 +20,9 @@
         "nyholm/psr7": "^1.3",
         "nyholm/psr7-server": "^1.0",
         "react/http": "^1.2",
-        "spiral/roadrunner": "^2.0",
+        "spiral/roadrunner": "^2.0 || ^2023.3 || ^2024.1",
```

Comment:
As I did understand @roxblnfk we should remove this requirement as we don't built on top of any specific roadrunner feature. 

So it is more:

```
spiral/roadrunner-http: ^2 || ^3.2
spiral/roadrunner-worker: ^2 || ^3
spiral/roadrunner-cli: ^2
```

@roxblnfk or @rustatian can one of you confirm this?

Reply:
Yep, you don't need any additional packages (jobs/kv/grpc/temporal, etc) to support RR HTTP plugin.
Just need to keep in mind, that HTTP is only 10% of the RR 😃

Reply:
```suggestion
```

Think then its easier to remove that requirement so we dont need always keep it uptodate.

Reply:
@msmakouz correct me if I wrong, but @alexander-schranz you can safely remove this, since that dep declared inside `spiral/roadrunner-http` dep.

Reply:
@alexander-schranz @rustatian Hi, yes, it makes sense, I will double-check and delete it.

Reply:
@alexander-schranz `spiral/roadrunner` removed.